### ISSUE TITLE
missingInnerMessage in Fluids.System is tool-specific.

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#549]
**Reported by stefanv on 11 May 2011 18:40 UTC**
The missingInnerMessage annotation in Modelica.Fluids.System says,

```
    missingInnerMessage="
Your model is using an outer \"system\" component but
an inner \"system\" component is not defined.
For simulation drag Modelica.Fluid.System into your model
to specify system properties. The default System setting
is used for the current simulation."
```

This implies that the tool will automatically create a System component if none was explicitly created by the user. However, this is at odds with the Modelica specification, which states that a tool<ins>may</ins> temporarily create such a component, not that it<ins>must</ins>.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/549

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 2 Aug 2012 13:30 UTC**
Stefan, how would you formulate this then instead?

**Comment by modelica-trac-importer** [comment]
**Comment by stefanv on 3 Aug 2012 19:53 UTC**
The easiest change would be to simply remove the last sentence.

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 3 Aug 2012 21:59 UTC**
Good idea. Fixed in r5373 and facf0f4ebe34d824b4d214704018c093a6f45813
